src/IO/InputStreamFromROOTFile.cxx

```diff
@@ -89,14 +98,36 @@ InputStreamFromROOTFile::set_up(const std::string & header_path)
 
     stream_ptr->SetBranchAddress("time1", &time1);
     stream_ptr->SetBranchAddress("time2", &time2);
-
+    stream_ptr->SetBranchAddress("axialPos",&axialPos);
```

Comment:
why is this here? Is it used at all? I can't find any reference to it. should it be in the list below of optional values?

Reply:
moved few lines below